src/strands/experimental/bidirectional_streaming/event_loop/__init__.py

```diff
@@ -1,15 +1,9 @@
 """Event loop management for bidirectional streaming."""
 
 from .bidirectional_event_loop import (
-    BidirectionalConnection,
-    bidirectional_event_loop_cycle,
-    start_bidirectional_connection,
-    stop_bidirectional_connection,
+    BidirectionalAgentLoop,
 )
 
 __all__ = [
-    "BidirectionalConnection",
-    "start_bidirectional_connection",
-    "stop_bidirectional_connection",
-    "bidirectional_event_loop_cycle",
+    "BidirectionalAgentLoop",
```

Comment:
do we actually want to export this? Is there any use case where customers would like to access agentic loop directly?